Jenkinsfile

```diff
@@ -69,12 +69,20 @@ pipeline {
         }
 
         stage('Build (Java)') {
+            environment {
+                VERSION = readMavenPom(file: 'parent/pom.xml').getVersion()
+            }
             steps {
                 container('maven') {
-                    configFileProvider([configFile(fileId: 'maven-nexus-settings-zeebe', variable: 'MAVEN_SETTINGS_XML')]) {
+                    configFileProvider([configFile(fileId: 'maven-nexus-settings-zeebe-local-repo', variable: 'MAVEN_SETTINGS_XML')]) {
```

Comment:
This change will be supported by https://github.com/camunda/infra-core/pull/1729 but I'm holding back merging https://github.com/camunda/infra-core/pull/1729 before it becomes clear that this PR get's green-lit.

Reply:
@cmur2 should that change be made before this is merged?

Reply:
The order here is arbitrary as the change is already applied in https://ci.zeebe.camunda.cloud/configfiles/editConfig?id=maven-nexus-settings-zeebe-local-repo so my PR https://github.com/camunda/infra-core/pull/1729 can go before or after, it merely makes official configuration as code what is applied in prod.